style.css

```diff
@@ -0,0 +1,74 @@
+body {
+  background-color: rgb(7, 7, 88);
+  display: flex;
+  flex-direction: column;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+h3 {
+  color: aliceblue;
+}
+
+.tools {
+  height: 15px;
+}
+
+.description {
+  font-size: 30px;
+  margin-left: 150px;
+  margin-right: 150px;
```

Comment:
- [ ] Please kindly ensure that the paragraph matches the design. You can remove the margins around the paragraph so it looks more like the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/158702872-507e2080-28f2-4976-aebf-b853a93ca281.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/158702932-6a3a10cb-8cdd-4bd4-b8f8-278aa290791e.png)